projects/cps-ui-kit/src/lib/components/cps-file-upload/cps-file-upload.component.ts

```diff
@@ -56,6 +67,18 @@ export class CpsFileUploadComponent implements OnInit, OnChanges {
   @Input() fileProcessingCallback?: (file: File) => Observable<boolean> =
     undefined;
 
+  /**
+   * Position of infoTooltip, it can be 'top', 'bottom', 'left' or 'right'.
+   * @group Props
+   */
+  @Input() infoTooltipPosition: CpsTooltipPosition = 'top';
+
+  /**
+   * Tooltip offset for styling.
+   * @group Props
+   */
+  @Input() tooltipOffset: number = 12;
```

Comment:
```suggestion
  /**
   * Position of file name tooltip, it can be 'top', 'bottom', 'left' or 'right'.
   * @group Props
   */
  @Input() fileNameTooltipPosition: CpsTooltipPosition = 'top';

  /**
   * File name tooltip offset for styling.
   * @group Props
   */
  @Input() fileNameTooltipOffset = 12;
```